packages/core/android/build.gradle

```diff
@@ -107,14 +107,13 @@ android {
       buildConfig = true
     }
   }
-  if (agpVersion.tokenize('.')[0].toInteger() < 8) {
-    compileOptions {
-      sourceCompatibility JavaVersion.VERSION_11
-      targetCompatibility JavaVersion.VERSION_11
-    }
-    kotlinOptions {
-      jvmTarget = JavaVersion.VERSION_11.majorVersion
-    }
+
+  compileOptions {
```

Comment:
what is the reason for this change?

Reply:
All versions of RN that we support can be compiled with Java 17, and I had compatibility problems after bumping the Kotlin version and updating Gradle tools versions. I think we can safely drop support for Java 11 at this point, but I am happy to re-check if you believe this could be an issue.

Reply:
so what is the error you are getting?